roles/tower_credentials/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
We shouldn't be adding travis here,  we can great a job in zuul

Reply:
sure, I'll remove the travis and update the PR